### PR TITLE
ref: Add ruff rules for builtins (A)

### PR DESCRIPTION
**Pull request by cbornet**
Add ruff rules for builtins (A)

### CODE REVIEW

File: src/backend/base/langflow/components/prototypes/RunnableExecutor.py [hunk 1]
```diff
@@ -130,8 +130,8 @@ async def build_executor(self) -> Message:
         self.status = status
         return result_value
 
-    async def astream_events(self, input):
-        async for event in self.runnable.astream_events(input, version="v1"):
+    async def astream_events(self, runnable_input):
```
Comment (cbornet) [inline src/backend/base/langflow/components/prototypes/RunnableExecutor.py:133]: Strictly speaking this is a breaking change. Is it acceptable here ?

File: src/backend/base/langflow/services/database/models/user/crud.py [hunk 2]
```diff
@@ -14,8 +14,8 @@ def get_user_by_username(db: Session, username: str) -> User | None:
     return db.exec(select(User).where(User.username == username)).first()
 
 
-def get_user_by_id(db: Session, id: UUID) -> User | None:
-    return db.exec(select(User).where(User.id == id)).first()
+def get_user_by_id(db: Session, user_id: UUID) -> User | None:
```
Comment (cbornet) [inline src/backend/base/langflow/services/database/models/user/crud.py:17]: Strictly speaking this is a breaking change. Is it acceptable here ?

File: src/backend/base/langflow/services/telemetry/opentelemetry.py [hunk 3]
```diff
@@ -64,13 +64,13 @@ def __init__(
         self,
         name: str,
         description: str,
-        type: MetricType,
+        metric_type: MetricType,
```
Comment (cbornet) [inline src/backend/base/langflow/services/telemetry/opentelemetry.py:67]: Strictly speaking this is a breaking change. Is it acceptable here ?

Comment (ogabrielluiz) [reply src/backend/base/langflow/services/telemetry/opentelemetry.py:67]: yes

**Review by ogabrielluiz — APPROVED** [review]
LGTM

**Pull request merged**